shared/lsp/registration.ts

```diff
@@ -389,33 +389,47 @@ export async function register({
         }
     }
 
+    function addRoot(root: sourcegraph.WorkspaceRoot): void {
+        const connectionPromise = (async () => {
+            try {
+                const serverRootUri = clientToServerURI(
+                    new URL(root.uri.toString())
+                )
+                const connection = await connect({
+                    clientRootUri: new URL(root.uri.toString()),
+                    initParams: {
+                        processId: null,
+                        rootUri: serverRootUri.href,
+                        capabilities: clientCapabilities,
+                        workspaceFolders: null,
+                        initializationOptions,
+                    },
+                    registerProviders: true,
+                })
+                subscriptions.add(connection)
+
+                connection.closeEvent.subscribe(() => {
+                    if (connectionsByRootUri.has(root.uri.toString())) {
+                        logger.log(
+                            'Refreshing WebSocket connection to language server'
+                        )
+                        addRoot(root)
+                    }
+                })
```

Comment:
```suggestion
                subscriptions.add(connection.closeEvent.subscribe(() => {
                    if (connectionsByRootUri.has(root.uri.toString())) {
                        logger.log(
                            'Refreshing WebSocket connection to language server'
                        )
                        addRoot(root)
                    }
                }))
```
I thought this would give an eslint warning... could you check if I need to fix the configuration?

Reply:
Updated. No lint.

Reply:
Sorry about that, I'll look into it